package.json

```diff
@@ -1,24 +1,100 @@
 {
   "name": "mockserver",
-  "version": "1.0.0",
-  "description": "A mockserver that allows you to mock XHR, long-polling XHR, server sent events and websocket connections",
-  "main": "index.js",
+  "version": "0.0.1-pre",
+  "description":
+    "A mockserver that allows you to mock XHR, long-polling XHR, server sent events and websocket connections",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node dist/index.js",
+    "prestart": "npm run build",
+    "build": "npm run build-ts",
+    "prewatch": "npm run build",
+    "watch":
+      "concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" -c \"cyan.bold,green.bold\"  \"npm run watch-ts\" \"npm run watch-node\"",
+    "watch-node": "nodemon dist/index.js",
+    "build-ts": "tsc",
+    "watch-ts": "tsc -w",
+    "setup:git-hooks": "./scripts/pre-install",
+    "commitlint": "commitlint -e",
+    "test": "jest",
+    "format":
+      "prettier --write .{/**,}/*.{md,js,ts} && prettier --write ./**/__tests__/*.{md,js,ts}"
   },
   "repository": {
     "type": "git",
     "url": "git+ssh://git@github.com/mesosphere/mockserver.git"
   },
-  "keywords": [
-    "mock",
-    "mockserver",
-    "testing"
-  ],
+  "keywords": ["mock", "mockserver", "testing"],
   "author": "DanielMSchmidt <dschmidt@mesosphere.com>",
   "license": "MIT",
+  "homepage": "https://github.com/mesosphere/mockserver#readme",
   "bugs": {
     "url": "https://github.com/mesosphere/mockserver/issues"
   },
-  "homepage": "https://github.com/mesosphere/mockserver#readme"
+  "jest": {
+    "projects": [
+      {
+        "displayName": "Jest",
+        "transform": {
+          "^.+\\.ts$": "ts-jest"
+        },
+        "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(js|ts)$",
+        "moduleFileExtensions": ["ts", "js", "json"]
+      },
+      {
+        "displayName": "Prettier",
+        "runner": "jest-runner-prettier",
+        "moduleFileExtensions": [
```

Comment:
🎨  Please remove the unnecessary bits to not pollute the package.json.

Reply:
done 👍